src/sentry/static/sentry/app/components/charts/components/legend.jsx

```diff
@@ -1,8 +1,17 @@
 import 'echarts/lib/component/legend';
+import 'echarts/lib/component/legendScroll';
 
 export default function Legend(props = {}) {
   return {
     show: true,
+    type: 'scroll',
+    formatter: function(name) {
```

Comment:
We'll want to change this in the follow up PR but we should have a property name that does not exist in `echarts` to handle truncation instead of having it always enabled. e.g. We would do:

```
<Chart
 legend={{truncate: 80}}
/>
```

Reply:
made a ticket for it!